src/main/java/com/conveyal/datatools/manager/jobs/validation/SharedStopsValidator.java

```diff
@@ -103,8 +110,8 @@ public void validate() {
                 }
 
                 // Check for SS_03 (stop_id referenced doesn't exist)
-                // TODO: CHECK FEED ID (adjust the pre-build constructor to include feed_id)
-                if (!stopIds.contains(stopId)) {
+                // Make sure this error is only returned if we are inside the feed that is being checked
+                if (feedId.equals(sharedStopFeedId) && !stopIds.contains(stopId)) {
                     registerError(NewGTFSError.forFeed(NewGTFSErrorType.SHARED_STOP_GROUP_ENTITY_DOES_NOT_EXIST, stopId));
```

Comment:
Can you have this error be reported for the current line (instead of the entire feed) of the shared stops file?

Reply:
The shared stops file is not in Postgres so the validator can't talk to it. the `stopId` links it to the stops file

Reply:
I was thinking of line numbers because the shared stops file is processed line by line using `configReader.readRecord()`.